src/locale/lang/en.json

```diff
@@ -11,9 +11,11 @@
     },
     "multiShopSelector": {
```

Comment:
D'une façon globale sur les langues : préviens Laure qu'il va falloir mettre en place un système de traduction des chaines. Minimum FR IT ES DE dans un 1er temps.